src/App.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import "./App.css";
+import Header from "./Header";
+import HeroesGrid from "./HeroesGrid";
+
+class App extends React.Component {
+  state = {
+    heroes: []
+  };
+
+  componentDidMount() {
+    this.getRequestInJSON("https://api.opendota.com/api/heroStats").then(
+      heroesGot => {
+        this.setState({
+          heroes: heroesGot,
+          allHeroes: heroesGot
+        });
+      }
+    );
+  }
+
+  getRequestInJSON = url => {
```

Comment:
This function will probably be used a lot right? In every request in every component, you will probably redefine this method. How about extracting it to a file with just responsibility? like `request.js`. Usually developers extract all related requests methods in a "HttpClient" file, so this file is just responsible to configuring how to make requests (when extra steps are needed to do the request)